### DISCUSSION TITLE
Does warning affects to grade?

### DISCUSSION THREAD

**Discussion by seo-rii** [ANLAB-KAIST/KENSv3#173] [Project Q&A 4. Etc] [answered]
I'm using `#pragma region` for code readability, but it makes unknown-pragmas warning on lower gcc version(<13).
Can I just ignore this, or should I remove it before submission?

**Comment by Pusnow** [comment, marked as answer]
As long as it compiles and passes all tests, warnings are okay.

BTW, we will use gcc (Ubuntu 9.4.0-1ubuntu1~20.04.2) 9.4.0 or similar to grade your submissions.
